Dockerfile

```diff
@@ -1,37 +1,16 @@
-FROM quay.io/ukhomeofficedigital/alpine:v3.14 as builder
-
-USER root
-
-RUN apk add --no-cache openjdk11-jre
+FROM quay.io/ukhomeofficedigital/hocs-base-image as builder
 
 COPY build/libs/*.jar .
 
 RUN java -Djarmode=layertools -jar *.jar extract
 
 FROM quay.io/ukhomeofficedigital/hocs-libreoffice
 
-USER root
-
-RUN apk add --no-cache openjdk11-jre
-
 WORKDIR /app
 
-ENV USER user_hocs
-ENV USER_ID 1000
-ENV GROUP group_hocs
-
-RUN addgroup -S ${GROUP} && \
-    adduser -S -u ${USER_ID} ${USER} -G ${GROUP} -h /app && \
-    mkdir -p /app && \
-    chown -R ${USER}:${GROUP} /app
-
 COPY scripts/run.sh /app/scripts/run.sh
 
-RUN chmod a+x /app/scripts/*
```

Comment:
Do we not need this and L33/34?

Reply:
No because we already have it checked in as executable, and lines 33/34 we don't have any snapshot depenedencies and it would be decision point to start using them, at the moment it's just copied from a tutorial.